doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java

```diff
@@ -243,23 +246,22 @@ private void addModuleFiles(
                 // -----------------------------------------------------------------------
                 // Handle key without case differences
                 // -----------------------------------------------------------------------
-                for (Map.Entry<String, DocumentRenderer> entry : files.entrySet()) {
-                    if (entry.getKey().equalsIgnoreCase(key)) {
-                        DocumentRenderingContext originalDocRenderingContext =
-                                entry.getValue().getRenderingContext();
+                String originalKey = caseInsensitiveFiles.put(key.toLowerCase(), key);
```

Comment:
lower case always goes with `Locale#ROOT` to avoid portability issues.

Reply:
I thought the purpose of this code was to flag potential file key conflicts _on the build system_ specifically handling Linux vs. Windows etc. So wouldn't it be good to flag a casing conflict for the build system's default locale? If I use root, it may not flag it for my particular laptop's build.  Easy to change if you don't agree, just genuinely curious.

Reply:
No, I would expect the system to work identically on the same OS regardless of the actual locally used. This we want to avoid: https://reproducible-builds.org/docs/locales/ to stuff like with `tr_TR` and the dotless I.